addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -360,10 +360,12 @@ export class AddPageTemplates extends Component {
         const activePaneEl = this.panesRef.el.querySelector(".active");
         activeTabEl?.classList?.remove("active");
         activePaneEl?.classList?.remove("active");
+        activePaneEl?.setAttribute("inert", true); // Make sure trapFocus() works.
```

Comment:
I did not know this attribute but it is a boolean attribute. It does not mean it can contain true or false but that it can be there or not. When it's there, the only valid values for boolean attribute are: no value, the empty string or the attribute name itself (here: `'inert'`).